File: app/src/main/java/com/amarnehsoft/holyquran/main/readersFragment/ReadersProvider.java
package com.amarnehsoft.holyquran.main.readersFragment;

import com.amarnehsoft.holyquran.test.Reader;

import java.util.ArrayList;
import java.util.List;

public class ReadersProvider {
    public static List<Reader> getReaders(){
        ArrayList<Reader> readers = new ArrayList<>();
        readers.add(Reader.mahermuaiqly);
        readers.add(Reader.alafasy);
        readers.add(Reader.ahmedajamy);
        readers.add(Reader.abdurrahmaansudais);
        readers.add(Reader.shaatree);
        readers.add(Reader.abdulsamad);
        readers.add(Reader.saoodshuraym);
        readers.add(Reader.husary);
        readers.add(Reader.husarymujawwad);
        readers.add(Reader.hudhaify);
        readers.add(Reader.hanirifai);
        readers.add(Reader.muhammadayyoub);
        readers.add(Reader.muhammadjibreel);
        readers.add(Reader.ibrahimakhbar);
        readers.add(Reader.abdullahbasfar);
        readers.add(Reader.parhizgar);
        return readers;
    }
}

File: app/src/main/java/com/amarnehsoft/holyquran/main/readersFragment/ReadersProvider.kt
package com.amarnehsoft.holyquran.main.readersFragment

import com.amarnehsoft.holyquran.test.Reader

import java.util.ArrayList

object ReadersProvider {
    val readers: List<Reader>
        get() {
            val readers = ArrayList<Reader>()
            readers.add(Reader.MaherMuaiqly)
            readers.add(Reader.Alafasy)
            readers.add(Reader.AhmedAjamy)
            readers.add(Reader.AbdurrahmaanSudais)
            readers.add(Reader.Shaatree)
            readers.add(Reader.Abdulsamad)
            readers.add(Reader.SaoodShuraym)
            readers.add(Reader.Husary)
            readers.add(Reader.HusaryMujawwad)
            readers.add(Reader.Hudhaify)
            readers.add(Reader.HaniRifai)
            readers.add(Reader.MuhammadAyyoub)
            readers.add(Reader.MuhammadJibreel)
            readers.add(Reader.IbrahimAkhbar)
            readers.add(Reader.AbdullahBasfar)
            readers.add(Reader.Parhizgar)
            return readers
        }
}
